ACadSharp.Tests/Entities/MTextValueTestData.cs

```diff
@@ -0,0 +1,494 @@
+﻿#nullable enable
+using System;
+using ACadSharp.Entities;
+using System.Collections.Generic;
+
+namespace ACadSharp.Tests.Entities
+{
+	public class MTextValueTestData
```

Comment:
This class should not be in the namespace `ACadSharp.Tests.Entities`. I think is better to keep things in order, this should be in the `Common`, `Data` or a `Factory` folder, this way we keep the Entities tests in this namespace and the testing tools in auxiliary ones.